code-challenges.test.js

```diff
@@ -22,29 +28,71 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe('isDivisibleByThree', () => {
+  it('should be divisible by three when given 15', () => {
+   expect(isDivisibleByThree(15)).toEqual("15 is divisble by three")
+  })
+  it('should be divisible by three when given 0', () => {
+   expect(isDivisibleByThree(0)).toEqual("0 is divisble by three")
+  })
+  it('should NOT be divisible by three when given -7', () => {
+   expect(isDivisibleByThree(-7)).toEqual("-7 is not divisible by three")
+  })
+})
 
 
 // b) Create the function that makes the test pass.
 
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+// First, I need to return a new instance of the array "nouns" and transform each noun
+// We can either build an array manually, or by using the Map method
+// We can then call toUpperCase() on each noun while iterating through the array
+
+const capitalized = (nouns) => {
+ return nouns.map(noun => noun.toUpperCase());
+};
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"];
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"];
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+describe('capitalized', () => {
+  it('should return an array of all capitalized words', () => {
+   expect(capitalized(randomNouns1)).toEqual(["STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW"])
+   expect(capitalized(randomNouns2)).toEqual(["TEMPERATURE", "DATABASE", "CHOPSTICKS", "MANGO", "DEDUCTION"])
+ });
+});
```

Comment:
It looks like you used a different expected outcome in your tests than was given in the prompt. The goal is to have the output match exactly.